rs/sns/governance/src/types.rs

```diff
@@ -369,6 +369,12 @@ impl NervousSystemParameters {
     /// hosting the SNS.
     pub const MAX_NUMBER_OF_PRINCIPALS_PER_NEURON_CEILING: u64 = 15;
 
+    /// This is a lower bound for `max_dissolve_delay_seconds`.
+    /// Decreasing it below this number is problematic because other canisters
+    /// assumes that there are allowed to be at least 5 principals per neuron
```

Comment:
> other canisters assumes [sic.] that there are allowed to be at least 5 principals per neuron

Please, be specific. For example,

> Swap needs this to be at least 5, because it does X, Y, and Z.

Reply:
Done!